static/app/components/events/interfaces/frame/stacktraceLink.spec.tsx

```diff
@@ -126,4 +126,22 @@ describe('StacktraceLink', function () {
       expect(container).toBeEmptyDOMElement();
     });
   });
+
+  it('should hide stacktrace link error state on unsupported platforms', async function () {
+    MockApiClient.addMockResponse({
```

Comment:
Does this mean that we will still hit the API but we will not show anything in the UI?
If that's the case, I'm happy about it because we will be able to gather metrics for all platforms in the API.

Reply:
yes exactly, so far we're only hiding the error/setup call to action when it fails on usupported platforms and minified js